providers/fab/src/airflow/providers/fab/auth_manager/fab_auth_manager.py

```diff
@@ -155,7 +155,7 @@
 
 _MAP_MENU_ITEM_TO_FAB_RESOURCE_TYPE = {
     MenuItem.ASSETS: RESOURCE_ASSET,
-    MenuItem.ASSET_EVENTS: RESOURCE_ASSET,
+    MenuItem.EVENTS: RESOURCE_ASSET,
```

Comment:
This is incorrect too. Let's make sure all the events menu items are linked to the audit log. We can name the menu item Audit Log if that removes the confusion

Reply:
Good catch! I like better "Events" but that's more a personal opinion I guess. We can keep "Events" for now and rename it later if that makes more sense to people